target-platform/target-platform.target

```diff
@@ -34,6 +30,9 @@
       <unit id="org.apache.poi.ooxml.schemas" version="0.0.0"/>
       <unit id="org.apache.logging.log4j.to.slf4j" version="0.0.0"/>
       <unit id="slf4j.simple" version="0.0.0"/>
+      <unit id="org.eclipse.collections.api" version="12.0.0"/>
+      <unit id="org.eclipse.collections.impl" version="12.0.0"/>
+      <unit id="org.apache.aries.spifly.dynamic.bundle" version="0.0.0"/>
```

Comment:
Should these really be consumed from nat table or are these more dependencies of this project and can be added e.g. as maven coordinates or consumed from the original update sites directly?

Reply:
For the 12.0.0 version there is no original update site. Seems that was missed and is only available for the 13.0.0. But of course I can use the maven coordinates.